build.gradle

```diff
@@ -7,12 +7,50 @@ description = 'RxJava: Reactive Extensions for the JVM – a library for composi
 
 apply plugin: 'rxjava-project'
 apply plugin: 'java'
+apply plugin: 'findbugs'
+apply plugin: 'jacoco'
 
 dependencies {
     testCompile 'junit:junit-dep:4.10'
     testCompile 'org.mockito:mockito-core:1.8.5'
 }
 
+////////////////////////////////////////////////////////////////////
+// to run findbugs: 
+//     ./gradlew findbugsMain
+// then open build/reports/findbugs/main.html
+////////////////////////////////////////////////////////////////////
+
+findbugs {
+  ignoreFailures = true
```

Comment:
Yep I support that too. Steps would probably be
* get findbugs plugin in so that anyone can run it, not part of default build
* review output from findbugs plugin, configure exclusions
* decide if we want it part of the default build

Setting failure coverage thresholds for jacoco would be trickier as there are a lot of paths in the codebase that are triggered only in quite unusual concurrent circumstances and are tricky to induce in unit tests. I have been missing this sort of tool when zeroing in on particular operators for the purposes of bug finding and contributing new code.